app/controllers/assessments_controller.rb

```diff
@@ -868,6 +868,7 @@ def valid_asmt_tar(tar_extract)
     tar_extract.each do |entry|
       pathname = entry.full_name
       next if pathname.start_with? "."
+      next if pathname.start_with? "PaxHeader"
```

Comment:
what does PaxHeaders stand for?

Reply:
could you add a comment on it? so that next time people who look at this code would know